engine/src/main/java/com/redhat/cloud/notifications/processors/slack/SlackRouteBuilder.java

```diff
@@ -1,90 +0,0 @@
-package com.redhat.cloud.notifications.processors.slack;
-
-import org.apache.camel.CamelExchangeException;
-import org.apache.camel.builder.RouteBuilder;
-import org.eclipse.microprofile.config.inject.ConfigProperty;
-
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-
-import java.io.IOException;
-
-import static com.redhat.cloud.notifications.Constants.API_INTERNAL;
-import static org.apache.camel.LoggingLevel.ERROR;
-import static org.apache.camel.LoggingLevel.INFO;
-
-@ApplicationScoped
-public class SlackRouteBuilder extends RouteBuilder {
-
-    public static final String REST_PATH = API_INTERNAL + "/slack";
-    public static final String SLACK_OUTGOING_ROUTE = "slack-outgoing";
-    public static final String SLACK_INCOMING_ROUTE = "slack-incoming";
-
-    private static final String SLACK_DIRECT_ENDPOINT = "direct:slack";
-    private static final String EXCEPTION_DIRECT_ENDPOINT = "direct:exception";
-    private static final String ERROR_MSG = "Slack message sending failed [orgId=${exchangeProperty.orgId}, " +
-            "historyId=${exchangeProperty.historyId}, webhookUrl=${exchangeProperty.webhookUrl}, " +
-            "channel=${exchangeProperty.channel}]\n${exception.stacktrace}";
-
-    @ConfigProperty(name = "notifications.slack.camel.maximum-redeliveries", defaultValue = "2")
-    int maxRedeliveries;
-
-    @ConfigProperty(name = "notifications.slack.camel.redelivery-delay", defaultValue = "1000")
-    long redeliveryDelay;
-
-    @ConfigProperty(name = "notifications.slack.camel.max-endpoint-cache-size", defaultValue = "100")
-    int maxEndpointCacheSize;
-
-    @Inject
-    SlackNotificationProcessor slackNotificationProcessor;
-
-    @Inject
-    RetryCounterProcessor retryCounterProcessor;
-
-    @Override
-    public void configure() {
-
-        /*
-         * An IOException can be thrown in case of network issue or of unexpected remote server (Slack) failure.
-         * It is worth retrying in that case. Retry attempts will be logged. The exception itself will also be
-         * logged eventually if none of the retry attempts were successful.
-         */
-        onException(IOException.class)
-                .maximumRedeliveries(maxRedeliveries)
-                .redeliveryDelay(redeliveryDelay)
-                .onRedelivery(retryCounterProcessor)
-                .retryAttemptedLogLevel(INFO)
-                .to(EXCEPTION_DIRECT_ENDPOINT);
-
-        /*
-         * We only saw a CamelExchangeException so far because of a misconfiguration of a Slack integration.
-         * Let's treat that like an HTTP 4xx error for now, which implies no retry. It will still be logged.
-         */
-        onException(CamelExchangeException.class)
-                .to(EXCEPTION_DIRECT_ENDPOINT);
```

Comment:
Because today the purpose `EXCEPTION_DIRECT_ENDPOINT` was just about adding more log details about the exception and nothing else, the global catch Exception on `CamelCommonExceptionHandler` will do the same job.
I don't think we need a route fot that yet, but maybe in the future.

Reply:
Perfect!